src/test/java/org/folio/innreach/controller/InnReachTransactionControllerTest.java

```diff
@@ -1807,6 +1807,41 @@ void cancelPatronHold_when_TransactionIsOnHoldOrTransfer_and_RequestIsClosed(Tra
 
     assertNotNull(updatedTransaction);
     assertEquals(TransactionStateEnum.BORROWING_SITE_CANCEL, updatedTransaction.getState());
+//    assertEquals(null,updatedTransaction.getHold().getFolioHoldingId());
+//    assertEquals(null,updatedTransaction.getHold().getFolioLoanId());
+//    assertEquals(null,updatedTransaction.getHold().getFolioInstanceId());
+//    assertEquals(null,updatedTransaction.getHold().getFolioItemId());
```

Comment:
Should these commented out items maybe be removed?

Reply:
These can be un-commented now, I have made the require change and committed back.